portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Hobbies</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <div id="content">
+    <h1>Hobbies</h1>
+
+    <h2>Sports</h2>
+
+    <h4>Tennis</h4>
+    <p>Tennis is easily defined as my favorite sport. I started playing with my father when I
+        was around four years old. He would take my brother and I out to play even though we were
+        not able to properly hit the ball over the net yet. Over the years, what used to be an 
+        activity meant to bring us together as a family transitioned into an activity I am
+        deeply passionate about.
+        <br><br>
+        I joined my first team and got my first coach as a freshman in high school. I joined
+        varsity right away and played at that level for all four years. Throughout my high school
+        career I earned many league awards and finished first or second in the league for three
+        consecutive years.
+        <br><br>
+        I still love tennis to this day, and I will continue practicing to participate in local 
+        tournaments.</p>
```

Comment:
`p` is short for paragraph, so it is more common to see it used to separate paragraphs of text (or `div`) and then have `div` wrap a blob of related content. Something like this:

I also change the formatting inside the `p` a bit, but I think yours likely still conforms to the style guide expect for not aligning all of the sentences... the first is -1 indented from the rest.

```
    <div class="hobby">
      <h4>Tennis</h4>
      <p>
        Tennis is easily defined as my favorite sport. I started playing with my father when I
        was around four years old. He would take my brother and I out to play even though we were
        not able to properly hit the ball over the net yet. Over the years, what used to be an 
        activity meant to bring us together as a family transitioned into an activity I am
        deeply passionate about.
      </p>
      <p>
        I joined my first team and got my first coach as a freshman in high school. I joined
        varsity right away and played at that level for all four years. Throughout my high school
        career I earned many league awards and finished first or second in the league for three
        consecutive years.
      </p>
      <p>
        I still love tennis to this day, and I will continue practicing to participate in local 
        tournaments.
      </p>
    </div>
```

Reply:
I aligned the sentences and reformatted my code as suggested. I switched a lot of my <br> into <p>, but for some paragraphs where I wanted it formatted in a specific I way I kept the <br> since it was easier than creating <p> for each list element (lines 82-97 of hobbies.html)